src/ui/InteractiveTutorial/InteractiveTutorialRoot.tsx

```diff
@@ -66,6 +82,47 @@ export function InteractiveTutorialRoot(): React.ReactElement {
       ),
       canNext: true,
     },
+    [iTutorialSteps.NSSelection as number]: {
+      content: (
+        <>
+          <Typography>The tutorial will adjust to your programming ability.</Typography>
+          <Typography>Bitburner has 2 types of scripts:</Typography>
+          <List>
+            <ListItem>
+              <Typography>NS1: Javascript from 2009, very simple. Recommended for beginners to programming.</Typography>
```

Comment:
Should NS1 really be recommended to anyone at this point? `Easier to start with but not recommended` is probably how I'd phrase this